src/main/java/com/minecolonies/entity/ai/citizen/farmer/Field.java

```diff
@@ -230,7 +230,7 @@ public Field(@NotNull ScarecrowTileEntity scarecrowTileEntity, InventoryPlayer p
                                          PLAYER_INVENTORY_HOTBAR_OFFSET
             ));
         }
-        calculateSize(world, location.down());
+        //calculateSize(world, location.down());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)